src/cf-forms/usage.md

```diff
@@ -130,26 +131,54 @@ Inputs should always be paired with a `label` for accessibility reasons.
 
 ### Basic text inputs
 
-<label class="a-label" for="text-input-example">A text input</label>
+<label class="a-label" for="textinput-example">A text input</label>
```

Comment:
I don't know that it matters all that much, it's only used to make clicking the label focus the input in the docs.